test/workerd/tests.mjs

```diff
@@ -36,14 +36,12 @@ export const url_parse = {
 export const buffer_implements = {
```

Comment:
I don't think this test in unenv makes any sense any more?

Reply:
It's not needed but I'm also not sure if removing it is the best course of action, since this tests that the CF workers has the following functions.

Reply:
> since this tests that the CF workers has the following functions.

It tests nothing else than workerd so I think test should live in workerd (where everything is probably already tested).

I fail to see what the current would ever catch that workerd test suite would not?

Reply:
That's true. I'll remove it.